test/probes/mongodb.test.js

```diff
@@ -113,6 +118,9 @@ function makeTests (db_host, host, isReplicaSet) {
     if (current.parent && !(current.parent.title in doThese)) {
       this.skip()
     }
+    if (current.title !== 'should distinct' && current.title !== 'should count') {
+      //this.skip();
+    }
 
```

Comment:
i could but didn't just because it's test code and not performance critical in anyway. it's easier to uncomment one line than three, or vice versa. call it laziness...